docs/application/tizen-studio/faq/faq.md

```diff
@@ -0,0 +1,180 @@
+         }
+         .css-accordion .accordion-item input[type=radio] ~ .item-content-container {
+         overflow: hidden;
+         }
+         .css-accordion .accordion-item input[type=radio] ~ .item-content-container .item-content {
+         height: 0;
+         transition: all 0.3s linear;
+         }
+         .css-accordion .accordion-item input[type=radio]:checked ~ .item-content-container {
+         height: auto;
+         overflow: visible;
+         }
+         .css-accordion .accordion-item input[type=radio]:checked ~ .item-content-container .item-content {
+         height: inherit;
+         /*! overflow-y: auto; */
+         transition: all 0.5s linear;
+         }
+         .css-accordion .accordion-item input[type=radio]:checked + label {
+         color: #008aee;
+         }
+         .css-accordion .accordion-item input[type=radio]:checked + label:before {
+         background: #f3f3f3;
+         }
+      
+      </style>
+   </head>
+<body>
+   <ol class="css-accordion user-journey">
+      <li class="accordion-item stage-1">
+         <input type="radio" name="accordion-control" id="stage-1-control" checked="">
+            <label for="stage-1-control">How to run Tizen Studio on Ubuntu 18.04?</label>
+               <div class="item-content-container">
+                  <div class="item-content">
+                     <p>Tizen studio supports Ubuntu 18.04. However, Ubuntu 18.04 distribution does not include the runtime library <b>libpng12-0</b>.
+                     <p>To experience optimal installation experience, install the runtime library and run the following command:
+                        <pre>
+                           $ wget http://mirrors.kernel.org/ubuntu/pool/main/libp/libpng/libpng12-0_1.2.54-1ubuntu1_amd64.deb
+                           $ sudo dpkg -i libpng12-0_1.2.54-1ubuntu1_amd64.deb
+                        </pre>
+                     </p>
+                     Generally, installing packages from older distributions may break apt installation system, but since these dependencies are also present in Ubuntu-18.04, so the apt installation system works flawlessly. </p>
+                  </div>
+               </div>
+      </li>
+      <li class="accordion-item stage-2">
+         <input type="radio" name="accordion-control" id="stage-2-control">
+            <label for="stage-2-control">How to manage certificates in different Ubuntu?</label>
+               <div class="item-content-container">
+                  <div class="item-content">
+                     <p>
+                      On Ubuntu, Tizen Certificate Manager in Tizen Studio stores the passwords of the certificates in the <b>gnome-keyring</b> application. Therefore, you must enable the <b>gnome-keyring</b> application to ensure that Tizen Studio functions smoothly. </p>
+                     <p>The remote login in a Linux desktop system does not have the <b>DBUS_SESSION_BUS_ADDRESS</b> variable set in the session. </p>
```

Comment:
The answer does not clarify how to manage on different Ubuntu versions?

Reply:
@nawab-samsung this one is not yet resolved.

Reply:
@nawab-samsung, please get a confirmation if the updated content addresses the concern.